internal/devbox/devbox.go

```diff
@@ -1033,21 +1033,28 @@ func (d *Devbox) PackageNames() []string {
 	return d.cfg.PackagesVersionedNames()
 }
 
-// ConfigPackages returns the packages that are defined in devbox.json
-// NOTE: the return type is different from devconfig.Packages
-func (d *Devbox) ConfigPackages() []*devpkg.Package {
+// AllPackages returns the packages that are defined in devbox.json and
+// recursively added by plugins.
+// * NOTE1: This will not return packages removed by their plugin with the
+// __remove_trigger_package field.
+// * NOTE2: the return type is different from devconfig.Packages
```

Comment:
optional: can remove NOTE2. It was there because the old function name was `ConfigPackages` which may have led a reader to assume the return type is `devconfig.Packages`